bin/node/node/src/main.rs

```diff
@@ -1,3 +1,19 @@
+// Copyright 2019-2020 Parity Technologies (UK) Ltd.
```

Comment:
Given that the license year starts in 2019 I think we need to change the `copyright_start_year` that's part of the CLI (line 35).